src/collective/lazysizes/profiles/plone5/registry.xml

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0"?>
+<registry
+    i18n:domain="plone"
+    xmlns:i18n="http://xml.zope.org/namespaces/i18n">
+
+  <!--
+    BUNDLES - ready to use
+    The compiled bundle files are empty - they are generated/cooked by Plone.
+  -->
+
+  <records prefix="plone.bundles/lazysizes" interface="Products.CMFPlone.interfaces.IBundleRegistry">
+    <value key="last_compilation"></value>
+    <value key="jscompilation">++plone++collective.lazysizes/bundle-lazysizes-compiled.min.js</value>
+    <value key="merge_with">default</value>
+    <value key="depends"></value>
+    <value key="compile">False</value>
+    <value key="enabled">True</value>
+    <value key="resources">
+      <element>lazysizes</element>
+    </value>
+  </records>
+
+  <!-- NOTE: if you use ``lazysizes-twitter``, disable the ``lazysizes`` bundle -->
```

Comment:
what do you mean with this?

Reply:
i made two bundles - lazysizes and lazysizes-twitter. the twitter one also loads lazysizes AFTER the twitter code. to not load two lazysizes, you have to disable one of them.

not so elegant, but maybe the simplest solution.

the twitter code should be loaded before lazysizes itself - so say the docs. with bundles you can only define dependencies, but no insert-before. we cannot let lazysizes depend on lazysizes-twitter, because the latter might just not be loaded (as it's the default). therefore how it's done here.

Reply:
but why do we want 2 different bundles? how do you select or the other? you need to explain better this not only for us, but for the rest of the integrators; a note on the README should be added in case we decide that 2 bundles are needed indeed.